scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/synchronization/SynchronizationWrapper.java

```diff
@@ -0,0 +1,256 @@
+            throws InvalidChannelException {
+        return internalAPI.getOrDefault(originator, taskId, channel, key, defaultValue);
+    }
+
+    @Override
+    public void forEach(String channel, String action) throws InvalidChannelException, CompilationException {
+        internalAPI.forEach(originator, taskId, channel, action);
+    }
+
+    @Override
+    public void replaceAll(String channel, String function)
+            throws InvalidChannelException, CompilationException, IOException {
+        internalAPI.replaceAll(originator, taskId, channel, function);
+    }
+
+    @Override
+    public Serializable putIfAbsent(String channel, String key, Serializable value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.putIfAbsent(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public boolean remove(String channel, String key, Serializable value) throws InvalidChannelException, IOException {
+        return internalAPI.remove(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public boolean replace(String channel, String key, Serializable oldValue, Serializable newValue)
+            throws InvalidChannelException, IOException {
+        return internalAPI.replace(originator, taskId, channel, key, oldValue, newValue);
+    }
+
+    @Override
+    public Serializable replace(String channel, String key, Serializable value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.replace(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public Serializable computeIfAbsent(String channel, String key, String mappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.computeIfAbsent(originator, taskId, channel, key, mappingFunction);
+    }
+
+    @Override
+    public Serializable computeIfPresent(String channel, String key, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.computeIfPresent(originator, taskId, channel, key, remappingFunction);
+    }
+
+    @Override
+    public Serializable compute(String channel, String key, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.compute(originator, taskId, channel, key, remappingFunction);
+    }
+
+    @Override
+    public Serializable merge(String channel, String key, Serializable value, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.merge(originator, taskId, channel, key, value, remappingFunction);
+    }
+
+    @Override
+    public PredicateActionResult conditionalCompute(String channel, String key, String predicate,
+            String thenRemappingFunction) throws InvalidChannelException, CompilationException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)